go.mod

```diff
@@ -3,6 +3,7 @@ module sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm
 go 1.12
 
 require (
+	github.com/aws/aws-sdk-go v1.23.4 // indirect
```

Comment:
I think this might be leftover from working on CAPA?

Reply:
yup! that's not supposed to be there